ios/src/TGMapViewController.mm

```diff
@@ -149,6 +149,47 @@ - (void)pickFeaturesAt:(CGPoint)screenPosition
     });
 }
 
+- (void)pickLabelAt:(CGPoint)screenPosition
+{
+    if (!self.map && !self.mapViewDelegate) { return; }
```

Comment:
I guess if the user hasn't provided a delegate there's no need to overhead tangram with feature selection, I will make sure this is something mentioned in the documentation of this interface.